core/trino-main/src/main/java/io/trino/server/InternalAuthenticationManager.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
Should we update docs that this is now required?

CC: @mosabua

Reply:
yes .. on it

Reply:
https://github.com/trinodb/trino/pull/12129